src/lib/y2network/connection_config/ip_config.rb

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+module Y2Network
+  module ConnectionConfig
+    class IPConfig
+      # @return [IPAddress] IP address
+      attr_accessor :address
+      # @return [Symbol] Address scope (:global, :site, :link or :host)
+      attr_accessor :scope
```

Comment:
what is this value in old implementation?

Reply:
I think it is out of scope and nothing we should deal with..

Reply:
OK, dropped.